package.json

```diff
@@ -1,51 +1,94 @@
 {
   "name": "evan-lang",
-  "version": "0.0.1",
-  "description": "A vaguely Lispy language in the context of JavaScript.",
+  "version": "0.1.0",
+  "description": "a vaguely lispy language in the context of js",
   "repository": {
     "type": "git",
     "url": "https://github.com/dslmeinte/evan-lang.git"
   },
+  "bin": {
```

Comment:
The central thing for `evan-lang` is that it "just" transforms any JSON, either according to sType or verbatim. The list containing the evaluation of `fib(0)`, ... is just a JSON list, not an sTyped thing so the expectation should be that it remains a list - that the list items transform according to an sType is irrelevant.
Doing an implicit `transform(list).last()` breaks that contract, so you'd have to introduce a new sType to make sure the listiness is preserved in case you do want the list. I think it's better to make sure that users can do `transform(list).last()` thing if they want it.